tests/functional/tests/management/test_disable_cleaner.py

```diff
@@ -0,0 +1,108 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+
+
+@pytest.mark.skip(reason="not implemented")
+def test_attach_cleaner_disabled(pyocf_ctx):
+    """
+    title: Attach cache with cleaner_disabled option set.
+    description: |
+      Check that cache can be attached when cleaner_disabled option is
+      selected and that "cleaning" metadata section is not allocated.
+    pass_criteria:
+      - Cache attaches properly.
+      - The "cleaning" metadata section is not allocated.
+      - Cache stops properly.
+    steps:
+      - Start the cache with default config.
+      - Prepare default attach config and set cleaner_disabled field to true.
+      - Attach cache device using prepared config.
+      - Verify that cache was attached properly.
+      - Verify that "cleaning" metadata section was not allocated.
```

Comment:
how? maybe we should check ~20% smaller metadata memory footprint?

Reply:
Currently the most reliable method is parsing the log. I wouldn't count on those 20%, as metadata contents may change in the future and this number may become different.